src/main/java/lotto/controller/LottoMachine.java

```diff
@@ -0,0 +1,83 @@
+package lotto.controller;
+
+import lotto.model.Lotto;
+import lotto.model.Reward;
+import lotto.model.WinningLogic;
+
+import java.util.*;
+
+import static lotto.model.Lotto.CANDIDATE_LOTTO_NUMBER_SIZE;
+import static lotto.model.Lotto.CORRECT_LOTTO_NUMBERS_SIZE;
+
+import static lotto.view.Input.*;
+import static lotto.view.Output.*;
+import static lotto.util.TypeConvert.*;
+
+public class LottoMachine {
+    public static Integer LOTTERY_PRICE = 1000;
+
+    public void purchaseStart() {
+        printStartMessage();
+        Integer purchaseAmount = inputMoneyForPurchase();
+        Integer purchased = countPurchasable(purchaseAmount);
+        printPurchasableMessage(purchased);
+        List<Lotto> bunchOfLotto = makeBunchOfLotto(purchaseAmount);
+
+        printBunchOfLottoNumbers(bunchOfLotto);
+        printInputWinningNumber();
+        WinningLogic winningLogic = new WinningLogic();
+        TreeMap<Integer,Integer> prizes = winningLogic.makePrizes(makeRewards(bunchOfLotto, makeWinningNumbers()));
+
+        printWinStatics();
+        print3Match(prizes.get(Reward.FIFTH_PRIZE.getPrizeMoney()));
+        print4Match(prizes.get(Reward.FOURTH_PRIZE.getPrizeMoney()));
+        print5Match(prizes.get(Reward.THIRD_PRIZE.getPrizeMoney()));
+        print6MatchWithBonus(prizes.get(Reward.SECOND_PRIZE.getPrizeMoney()));
+        print6Match(prizes.get(Reward.FIRST_PRIZE.getPrizeMoney()));
+
+        printYield(winningLogic.makeYield(purchaseAmount, winningLogic.makePrizeMoney(prizes)));
+    }
+
+    private List<Reward> makeRewards(List<Lotto> bunchOfLotto, List<Integer> winningNumbers) {
+        List<Reward> rewards = new ArrayList<>();
+        WinningLogic winningLogic = new WinningLogic();
+        for (Lotto lotto : bunchOfLotto) {
+            rewards.add(Reward.getReward(winningLogic.makeWinningLotto(lotto.getLottoNumbers(), winningNumbers)));
```

Comment:
```suggestion
            rewards.add(Reward.getReward(winningLogic.makeWinningLotto(lotto, winningNumbers)));
```
makeWinningLotto() 파라미터로  lotto 내부의 필드 값을 꺼내서 전달하는 것이 아니라  ``Lotto 객체를 전달``해주는 것이 어떨까요 😉